docs/src/howto-guides/008_config_local_mqtt_bind_address_and_port.md

```diff
@@ -0,0 +1,107 @@
+# How to configure the local port and bind address in mosquitto?
+
+Configuring a mosquitto port and bind address in thin edge is a three step process.
+
+> Note: The mqtt.port and the mqtt.bind_address can be set/unset independently.
+
+## Step 1: Disconnect the thin edge device
+
+The thin edge device has to be disconnected from the cloud using the `tedge` command
+
+```shell
+tedge disconnect c8y/az
+```
+
+## Step 2: Set and verify the new mqtt port and bind address
+
+Use the `tedge` command to set the mqtt.port and mqtt.bind_address with a desired port and bind address as below.
+
+```shell
+tedge config set mqtt.port 1024
+```
+
+```shell
+tedge config set mqtt.bind_address 127.0.0.1
+```
+
+> Note: The bind_address is the address of the one of the device interface.
+  For example this can be get as `ifconfig | grep inet` or set it to `0.0.0.0`
+
+This will make sure that all the mqtt clients use the newer port and the bind address that
+has been set once the device is connected to the cloud as in step 3.
+
+## Verify the port and the bind address configured/set
+
+Use the `tedge` command to print the mqtt port and bind address that has been set as below.
+
+```shell
+tedge config get mqtt.port
+```
+
+```shell
+tedge config get mqtt.bind_address
+```
+
+## Step 3: Connect the thin edge device to cloud
+
+Use the `tedge` command to connect to the desired cloud as below.
+
+```shell
+tedge connect c8y
+
+#or
+
+tedge connect az
+```
+
+This will configure all the services (mosquitto, tedge-mapper-c8y.service, tedge-mapper-az.service,
```

Comment:
I just got a question while reading this sentence. We also have `tedge-mapper-collectd.service`. The "all" includes this service or not?